README.md

```diff
@@ -102,14 +102,25 @@ For installation, prerequisites include sqlite3 and rvm. [Click here for a compl
     * You may need to first run `bundle install` if you have older gems in your environment from previous Rails work. If you get an error message like `Your Ruby version is 2.x.x, but your Gemfile specified 2.6.6` then you need to install the ruby version 2.6.6 using `rvm` or `rbenv`.
 	    * Using **rvm**: `rvm install 2.6.6` followed by `rvm use 2.6.6`
 	    * Using **rbenv**:  `rbenv install 2.6.6` followed by `rbenv local 2.6.6`
+    * If `bundle install` gives you following error 
+      `Your bundle is locked to mimemagic (0.3.3), but that version could not be found in any of the sources listed in your Gemfile. If you haven't changed sources, that means the author of mimemagic (0.3.3) has removed it. You'll need to update your bundle to a version other than mimemagic (0.3.3) that hasn't been removed in order to install`
+      **Delete `Gemfile.lock` and re run `bundle install`**
     * Run this `bundle config set without 'production mysql'` from the rails root folder to set your project to exclude libraries only needed in production.
     * Install gems with `bundle install` from the rails root folder.
 6. Run `cp db/schema.rb.example db/schema.rb` to make a copy of `db/schema.rb.example` in `db/schema.rb`.
 7. You could choose to use mysql2 or sqlite3 as your database
     * If mysql2,  run `cp config/database.yml.mysql.example config/database.yml` to make a copy of `config/database.yml.mysql.example` in `config/database.yml` 
     * If sqlite3, run `cp config/database.yml.sqlite.example config/database.yml` to make a copy of `config/database.yml.sqlite.example` in `config/database.yml`.
 _kindly note if you choose to use sqlite some tests may fail. The project was setup initially to use mysql and some tests are tailored for mysql db. No need for alarm, we are working to fix these and this will not interfere with your development process_
-8. Run `rake db:setup` to set up the database
+8. Run `rake db:setup` to set up the database. If this gives you error for `manifest.js` **missing**. Create your manifest file by using these steps
```

Comment:
Hey @achhetr . We added this installation step https://github.com/publiclab/plots2/commit/ea084aaea7fad65c2d05aff527c1ceb8f2ec5b44  that fixes the manifest.js error...I believe this was not there when you were installing, with this one shouldn't get this error..maybe we could remove this and also the changes on app/assets/config/manifest.js  and we can have this merged. What do you think?

Reply:
Hi @cesswairimu, I tried reinstallation after deleting folder using `rm -rf plot2`. After cloning I pull latest update. I got the same manifest.js error. I suggest to keep this error until the error persists.

Reply:
@achhetr did you run this ` $ rails webpacker:install && rails webpacker:install:react && rails generate react:install` during your fresh installation

Reply:
@cesswairimu apologies, I made a mistake by not merging update to the branch. It works fine. I will update `README.md` file.

Reply:
@cesswairimu I have updated the pull request. Thank you

Reply:
@achhetr, thanks for confirming

Reply:
Looks like other commits were touched during the update...rebasing should fix this. please let me know if you need help with this. Thanks again

Reply:
> Looks like other commits were touched during the update...rebasing should fix this. please let me know if you need help with this. Thanks again

Hey @achhetr , just checking what you think of this? thanks